sql/core/src/main/scala/org/apache/spark/sql/sources/interfaces.scala

```diff
@@ -549,15 +549,18 @@ class HDFSFileCatalog(
 
   refresh()
 
+  private def isInvalidFile(f: FileStatus): Boolean =
+    f.getPath.getName.startsWith("_") || f.getLen == 0
```

Comment:
I wanna filter out them earlier, so I choose here. Is empty file valid in some cases?